src/mk/def_rules.mk

```diff
@@ -1025,9 +1025,9 @@ endef
 define TABLE_TO_FILE_RECIPE = 
 	$(FILE_VARIABLES)
 	$(eval d = $($@_d))
-	@$(call echo_cmd,"=== Creating SQL TABLE from Sql statement [$(notdir $<)] in $(OBJLIB)")
+	@$(call echo_cmd,"=== Creating SQL TABLE $(OBJLIB)/$(basename $(notdir $@)) from Sql statement [$(notdir $<)]")
```

Comment:
For consistency, shouldn't we update all the other recipes below to also echo this same information?

Reply:
agreed

Reply:
Addressed this in my latest commit